python/lsst/analysis/drp/dataSelectors.py

```diff
@@ -151,3 +139,53 @@ def __call__(self, df, **kwargs):
         sourceType = np.zeros(len(df))
         sourceType[gals] = 2
         return sourceType
+
+
+class UnknownIdentifier(DataFrameAction):
+    """Identifies un classified objects from the dataFrame and marks them as a
+       9 in the added sourceType column"""
```

Comment:
Where does 9 come from? Do we have to use a numeric value, or could we use "UNKNOWN" or something that's easier to avoid misinterpreting?